This repository contains files pertaining to the manuscript "Genomic sureillance of SARS-CoV-2 omicron variants on a university campus" by Weil AA et al.

The delta_3A, hct_3B, and omicron_3C folders contain .json files for the phylogenetic trees included as figures in the manuscript. These trees can be viewed using the auspice.us website.

The sequence_information folder contains a list of GISAID virus names for all samples sequenced for this project.

The code file contains code used in the genomic analyses done for this manuscript:
"Snakefile" - A snakefile containing the series of nextstrain subcommands used to generate the delta, hct, and omicron trees. 
"Pairwise.py" - Python code used to calculate pairwise (aka hamming) distances between pairs of genomes.
"Snakefile_Traits" - A snakefile containing the series of nextstrain subcommands used to generate the phylogenetic tress that were then used to estimate the number of introductions of delta and omicron viruses onto campus represented by the sequenced study samples. The output .json from this series of subcommands was used as the input for IntroNum.py.
"IntroNum.py" - Python code used to calculate the number of variant introduction events observed in the trees generated by the "Snakefile_Traits" subcommands.
